components/WalletBindingSuccessController.tsx

```diff
@@ -0,0 +1,17 @@
+import {useContext} from 'react';
+import {GlobalContext} from '../shared/GlobalContext';
+import {useSelector} from '@xstate/react';
+import {VcEvents, selectWalletBindingSuccess} from '../machines/vc';
+
+export const UseWalletBindingSuccess = () => {
+  const {appService} = useContext(GlobalContext);
+  const vcService = appService.children.get('vc');
+  const isBindingSuccess = useSelector(vcService, selectWalletBindingSuccess);
```

Comment:
this selector `selectWalletBindingSuccess` expects context to have this `walletBindingSuccess`. But you have removed this from both the machine existing and esignet. 
Now how will component know about wallet binding has been done successfully, hence show a banner?